### ISSUE TITLE
Add tooltip to icons for invalid content (in wizard and in grid)

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#690]
1. Open new  folder-wizard.
2. Move cursor to `toolbar-state-icon`

![image](https://user-images.githubusercontent.com/3728712/61874327-cad74380-aef0-11e9-8fcc-c468d4fedf7f.png)

**Expected**: 'Invalid content' message should appear

The same issue in the grid:
![image](https://user-images.githubusercontent.com/3728712/61874472-29042680-aef1-11e9-94b3-9d4b24078626.png)

**Comment by alansemenov** [comment]
Use the existing i18n resource `field.invalid`

**Comment by ashklianko** [comment]
I believe we need an actual and precise description where we want to see the tooltips for the status icons, this description is outdated and doesn't cover the new UI design

**Comment by alansemenov** [comment]
> I believe we need an actual and precise description where we want to see the tooltips for the status icons, this description is outdated and doesn't cover the new UI design

Yes, we need all three workflow icons (in progress, ready, invalid) to display tooltips wherever they are shown. We even already have i18n keys for the other two: `tooltip.state.in_progress` and `tooltip.state.ready`
